go/inst/instance.go

```diff
@@ -121,13 +121,31 @@ type Instance struct {
 	LastDiscoveryLatency time.Duration
 
 	seed bool // Means we force this instance to be written to backend, even if it's invalid, empty or forgotten
+
+	/* All things Group Replication below */
+
+	// Group replication global variables
+	ReplicationGroupName            string
+	ReplicationGroupIsSinglePrimary bool
+
+	// Replication group members information. See
+	// https://dev.mysql.com/doc/refman/8.0/en/replication-group-members-table.html for details.
+	ReplicationGroupMemberState string
+	ReplicationGroupMemberRole  string
+
+	// List of all known members of the same group
+	ReplicationGroupMembers InstanceKeyMap
+
+	// Primary of the replication group
+	ReplicationGroupPrimaryKey InstanceKey
```

Comment:
Because `PrimaryKey` is a term in databases, let's use `PrimaryInstanceKey`
```suggestion
	ReplicationGroupPrimaryInstanceKey InstanceKey
```

Reply:
Fixed